ggj25/scripts/dodge_bubbal.gd

```diff
@@ -24,41 +17,21 @@ func _on_hud_start_game() -> void:
 
 # Called when the node enters the scene tree for the first time.
 func _ready() -> void:
+	super._ready()
 	game_timer.wait_time = GAME_TIME
 	hud.set_timer(str(GAME_TIME))
 	hud.setup_game()
-	ducks = find_ducks()
-	enable_duck_input(false)
 	
 	
 
 # Called every frame. 'delta' is the elapsed time since the previous frame.
 func _process(delta: float) -> void:
+	super._process(delta)
 	# xD this is the same code as in bubball.gd
```

Comment:
xD does this comment suggest this should go super too?

Reply:
It probably should, though at this point the "Generic" script is way too specific to be reused easily so will try something else